include/stout/flags/v1/positional_argument.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package stout.v1;
+
+// For defining our own options.
+import "google/protobuf/descriptor.proto";
+
+// Message which represents a positional argument.
+// NOTE: a positional argument is a bare value, and
+// its position in a list of arguments identifies it.
+// For example:
+//     echo 'hello' ' ' 'world'
+// 'hello', ' ', 'world' - are the positional args
+// from the command line.
+//
+// In our implementation of flags we expect that this
+// arguments will be passed without any name.(We added
+// 'repeated' string names and deprecate_names for
+// printing out the help message, obviously other users
+// will not use them).
+// Positional arguments might be followed in a random
```

Comment:
They're not in a random order, right? They're in a strict order.

e.g. if I define a command like `rename [source] [dest]`, I can't re-order those args: `source` must be first and `dest` must be second.